github_connector_odoo/models/github_organization.py

```diff
@@ -14,3 +14,31 @@ class GithubOrganization(models.Model):
     default_author_text = fields.Char(string="Default Author Text")
 
     runbot_url_pattern = fields.Char(string="Runbot URL Pattern")
+
+    create_series = fields.Boolean(
+        string="Auto Create Series",
+        help="If checked, during synchronization for branches"
+        " for which no series have been created,"
+        "they will be created automatically.",
+    )
+
+    fetch_branches = fields.Boolean(
+        string="Fetch all branches",
+        help="If checked, Fetch all repo branches even if no series are defined",
+    )
+
+    odoo_versions = fields.Text(
+        default="5.0, 5, 6.0, 6.1, 6, 7.0, 7, 8.0, 8, 9.0, 9, 10.0, 10, 11.0, 11, "
+        "12.0, 12, 13.0, 13, 14.0, 14, 15.0, 15, 16.0, 16",
+        required=True,
+    )
+
+    def update_odoo_versions(self):
+        """Check that the latest odoo version is in odoo_versions"""
+        repo_name = "odoo/odoo"
+        gh_api = self.get_github_connector()
+        odoo_repo = gh_api.get_repo(repo_name)
+        default_odoo_branch = odoo_repo.default_branch
+        if default_odoo_branch not in self.odoo_versions:
```

Comment:
Odoo versions are global. No need to keep them for each organisation separately

Reply:
resolved